README.md

```diff
@@ -106,14 +106,13 @@ For details on how to compare two arbitrary revisions, refer to the `integration
 
 ## Releases
 
-* The master branch is the current release and is considered stable at all
-  times.
-* New versions can be released as frequently as every two weeks (our sprint
-  cycle). The only exception would be for a high priority patch.
-* New releases occur the day after the sprint finishes.
-  * The version in the dev branch is updated
-  * The changes are merged into master
-  * A new tag is created
+There is a daily series of tasks run by Airflow (see the
+[`probe_scraper` DAG](https://github.com/mozilla/telemetry-airflow/blob/master/dags/probe_scraper.py))
+that uses the `master` branch of this repository as input and ends up pushing
+final JSONSchema and BigQuery schema files to the `generated-schemas` branch.
+At the time of writing, applying the new schemas still requires manual intervention
```

Comment:
This means that we can still expect existing schemas to be updated daily, right?  It would be good to clarify that here because from my outside perspective it's a little unclear what applying a new schema means.

Reply:
Perhaps `s/applying/deploying/` would make this less confusing. See also https://bugzilla.mozilla.org/show_bug.cgi?id=1609738.